### ISSUE TITLE
failed to define function flycheck-mode

### ISSUE THREAD

**Issue by anthrolisp** [dangom/org-thesis#2]
I get the following error when I load your sample thesis.org or manuscript.org

`File mode specification error: (error Autoloading file /home/username/.emacs.d/straight/build/org/org.elc failed to define function flycheck-mode)`

and the following error when I try to export it

`run-hooks: Autoloading file /home/username/.emacs.d/straight/build/org/org.elc failed to define function flycheck-mode`

I'm using your org-init.el as my only emacs config file.

**Comment by dangom** [comment]
Thanks for reporting, and sorry for not answering earlier. I suppose I missed the notification.
Did you get this issue fixed? Otherwise, could you please tell me your OS and Emacs version, and how you started Emacs?

I don't use the org-init.el as is, I just extracted it from my init to illustrate what my config looks like, but I'd of course be happy if it'd work as a standalone.

**Comment by anthrolisp** [comment]
Yes, still getting this error upon loading the sample org documents.
Here's my OS and Emacs versions: 

- Ubuntu v 19.04
- Emacs v 26.1

Many thanks in advance!
